### PR TITLE
Feat(web-react): Introduce Heading component (refs #DS-167)

### PR DESCRIPTION
**Pull request by literat**

### CODE REVIEW

File: packages/web-react/src/components/Heading/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+```jsx
+<Heading elementType="h1" size="large" />
+```
+
+## Available props
+
+| Name         | Type                                          | Description      |
+| ------------ | --------------------------------------------- | ---------------- |
+| `elementTyp` | `React.Element`, default `div`                | HTML tag         |
```
Comment (adamkudrna) [inline packages/web-react/src/components/Heading/README.md:13]:
```suggestion
| `elementType` | `React.Element`, default `div`                | HTML tag         |
```

File: packages/web-react/src/components/Heading/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+| `size`       | `xlarge`, `large`, `medium`, `small`, `xmall` | Size of the text |
```
Comment (adamkudrna) [inline packages/web-react/src/components/Heading/README.md:14]: Maybe a column with default value?

File: packages/web-react/src/types/heading.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { ElementType, JSXElementConstructor } from 'react';
+import { ChildrenProps } from './shared';
+
+export type TextBodySize = 'large' | 'medium' | 'small';
+export type HeadingSize = TextBodySize | 'xlarge' | 'xsmall';
```
Comment (adamkudrna) [inline packages/web-react/src/types/heading.ts:5]: I'm not sure if this is intended relation or just a coincidence of overlapping scales…

**Review by adamkudrna — APPROVED** [review]
Nice!

File: packages/web-react/src/types/heading.ts (hunk 3 above)
Comment (literat) [reply packages/web-react/src/types/heading.ts:5]: My intention is to define `shared/typography.ts` types after #228 will be merged and use `TextSize` as a default type extended by `xlarge` size.

**Pull request merged**